http/src/request/get_gateway_authed.rs

```diff
@@ -1,6 +1,10 @@
 use crate::request::prelude::*;
 use twilight_model::gateway::connection_info::BotConnectionInfo;
 
+/// Get information about the gateway through the lens of a bot.
```

Comment:
hmm, perhaps lens isn't the best choice for describing what this is? I had to do a double take to know what it meant.